com.unity.netcode.gameobjects/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Unity.Netcode
 {
-    internal enum NetworkPrefabOverride
+    public enum NetworkPrefabOverride
```

Comment:
**Comment:** I think this needs xmldocs

Reply:
to be completely honest, I've never actually understood the difference between these values. I'll see if I can find someone to explain it so I can write an xmldoc...